### PR TITLE
Add insufficient data and ready to train states to test model page

### PR DESCRIPTION
**Pull request by microbit-grace**
Part of [[Test model] Insufficient data, fist time training, retrain states](https://microbit-global.monday.com/boards/1356069004/pulses/1357714778) task

### CODE REVIEW

File: src/components/TrainModelFirstTitle.svelte [hunk 1]
```diff
@@ -18,15 +21,20 @@
...
-  <div class="flex gap-5">
-    <StandardButton onClick={() => navigate(Paths.DATA)}>
-      {$t('content.model.addData')}
-    </StandardButton>
-    <TrainingButton type="secondary" onClick={() => navigate(Paths.TRAINING)} />
+  {#if !sufficientData}
+    <p>
+      {$t('content.model.notEnoughDataInfoBody')}
+    </p>
+  {/if}
```
Comment (microbit-robert) [inline src/components/TrainModelFirstTitle.svelte:30]: Can we wrap these paragraphs in a div with space-y-5 and lose the mb-5? With sufficientData we only show the first paragraph. For some reason the margin doesn't collapse so we get a lot of vertical space between text and button.

image

**Review by microbit-robert — CHANGES_REQUESTED** [review]
Looks good. Minor change requested.

File: src/components/TrainModelFirstTitle.svelte (hunk 1 above)
Comment (microbit-grace) [reply src/components/TrainModelFirstTitle.svelte]: Makes sense!

Comment (microbit-robert) [reply src/components/TrainModelFirstTitle.svelte]: [10b2dd9](https://github.com/microbit-foundation/ml-trainer/pull/135/commits/10b2dd95234b6fc94bb36a1134645e8ac60a9c97)

**Review by microbit-robert — APPROVED** [review]
LGTM, thanks.

**Pull request merged**